### DISCUSSION TITLE
Changing packet rate to 333hz results in no input in INAV.

### DISCUSSION THREAD

**Discussion by mainframecn** [ExpressLRS/ExpressLRS#2662] [Q&A]
When I set my packet rate to 333hz, INAV doesn't seem to get any input from the receiver. The TX & RV do connect. Change it back to i.e. 250hz it works again. Am I missing some setting somewhere or could it be a limitation on the receiver? (i have updated both TX & RX to see it that makes a difference)
The goals is to get 16 (full) channels

Radio is TX16S M2, receiver is ATOMRC 2.4G ELRS and configured with CRSF in INAV.

**Comment by mmosca** [comment]
What is inav version and fc?
I fly all my INAV craft in 333Hz.
What about elrs versino on radio and rx?

**Reply by mainframecn** [reply]
Thanks for the response. 
Both TX and RX have ERLS v3.3.2. The FC is running on INAV 6 (maybe 6.1).  It probably has to do with the compatibility of the receiver. I'm still a learner in the RC world, i read on the ELRS site;

```
Supported RF Modes

SPI receivers DO NOT support D(D250, D500), F(F500, F1000) and Full Res(100Hz Full Res, 333Hz Full Res) Modes (Packet Rates) and thus will not bind or sync with a TX module in any of these modes.
```
So my guess is that the ATOMRC 2.4 RX module is an SPI receiver, although I haven't been able to confirm that on the specsheet of the module.

**Reply by mmosca** [reply]
INAV does not support SPI elrs. So I would be surprised if that was the reason.

As mentioned, I also run 333Hz on all my INAV builds without issue.

And just to make double sure, I just re-tested with my radio master pocket radio and a betafpv superp dual 2.4ghz receiver and attempted to change packet radios while connected and it still kept working.

I wouldn't discard the possibility of an issue in INAV caused by hardware issues, but I can't reproduce it. It may be worth checking if restarting INAV or wiggling/moving wires on your rx/fc connection could have any effects on this. Maybe your serial link between the fc and rx is introducing some errors or the wire is intermitent and  INAV is not handling that well.

**Reply by mmosca** [reply]
Also, try updating to INAV 7.1 just to see if it is an issue that has been fixed already
